lib/prune-publish.js

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2022 Andre 'Staltz' Medeiros <contact@staltz.com>
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+module.exports = function prunePublish(ssb, content, cb) {
+  const group = content.tangles.group
+
+  ssb.db.create({ content, encryptionFormat: 'box2' }, (err, msg) => {
+    if (err) {
+      if (group.previous.length > 1) {
+        const halfLength = Math.ceil(group.previous.length / 2)
+        group.previous = group.previous.slice(0, halfLength)
```

Comment:
random request - can you shuffle the previous before slicing?

This means that if we hit problems here different messages which converge the tangles differently and it _should_ converge faster. If you slice and always take the first 63 it might be slow...

Reply:
```suggestion
        group.previous = group.previous
          .sort((a, b) => Math.random() < 0.5 ? -1 : 1)
          .slice(0, halfLength)
```

Reply:
It gets shuffled in this PR in lib/add-group-tangle.js